examples/async-examples.ts

```diff
@@ -0,0 +1,53 @@
+import {RevAiApiClient, RevAiJobOptions} from 'revai-node-sdk';
+
+/* Initialize your client with your revai access token */
+const token = <YOUR-ACCESS-TOKEN>;
+var client = new RevAiApiClient(token);
+
+/* Get account details */
+var account = await client.getAccount();
+console.log(`Account: ${account.email}`);
+console.log(`Balance: ${account.balance_seconds} seconds`);
+
+/* Submit job and receive job object
+ * Media may be submitted from a local file or from a url by using:
+ * client.submitJobUrl("https://www.rev.ai/FTC_Sample_1.mp3");
+ */
+var jobOptions = new RevAiJobOptions(
+	metadata = "This is an example",
+	skip_diarization = true
+	);
+var job = client.submitJobLocalFile("./example.raw");
```

Comment:
@kbridbur I would like to make another case for `submitJobFromStream(stream)`.

`client.submitJobFromStream(fs.createReadStream('./example.raw'))'`

Reply:
@menioa 
Try running the example: I don't think it will work. Missing an `await`.
```suggestion
var job = await client.submitJobLocalFile("./example.raw");
```